ci-operator/config/openshift/telemeter/openshift-telemeter-master__okd-scos.yaml

```diff
@@ -21,6 +21,7 @@ images:
   inputs:
     ocp_builder_rhel-9-golang-1.21-openshift-4.16:
       as:
+      - "0"
```

Comment:
:flags:

Reply:
Stages are implicitly named "0", "1", ... You can see the Dockerfile copying from those stages: https://github.com/openshift/telemeter/blob/master/Dockerfile `--from=0` . Should be harmless to merge, but I will update the logic to leave out unnamed stages.